src/render_cmd.rs

```diff
@@ -260,10 +265,9 @@ impl RenderCmd {
         //
         // The issue is reported and open https://github.com/helm/helm/issues/8268
         //   as of 2020-07-26
-
         let result = Exec::shell(cmd)
             .stdout(Redirection::Pipe)
-            .stderr(Redirection::Merge)
+            .stderr(Redirection::Pipe)
```

Comment:
Great idea. I'll give the branch another spin.

Reply:
Cool, it works as expected.

I'd propose that we also output `stderr` to the console for debug log-level.